src/pages/cluster-management/cluster-configuration.html

```diff
@@ -0,0 +1,143 @@
+                        <div class="form-group row">
+                            <label for="node-rpc-address" class="col-lg-5 col-xl-5 col-form-label">{{'cluster_management.cluster_page.node_address' | translate}}</label>
+                            <div class="col-lg-7 col-xl-7" tooltip="{{'cluster_management.cluster_page.node_address_tooltip' | translate}}">
+                                <input class="form-control" placeholder="{{'cluster_management.cluster_page.node_address_placeholder' | translate}}"
+                                       type="text" ng-model="rpcAddress" id="node-rpc-address">
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="col-xs-12">
+                                <button type="button" class="btn btn-primary addNodeBtn"
+                                        ng-click="addNodeToList(rpcAddress)" ng-disabled="!rpcAddress"
+                                        tooltip="{{'cluster_management.cluster_page.add_node_btn_tooltip' | translate}}"
+                                        tooltip-placement="top">
+                                    <span class="icon-plus"></span>
+                                    {{'cluster_management.cluster_page.add_node_btn' | translate}}
+                                </button>
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="col-xs-12">
+                                <div ng-if="clusterConfiguration.nodes.includes(rpcAddress)" class="nodeAlreadyAdded alert alert-danger mt-1">
+                                    {{'cluster_management.cluster_page.errors.node_in_list' | translate}}
+                                </div>
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="col-xs-12 mb-1 mt-1">
+                                <h5>{{'cluster_management.cluster_page.cluster_nodes_list' | translate}}</h5>
+                                <div class="row">
+                                    <div class="col-xs-12">
+                                        <div ng-if="clusterConfiguration.nodes.length < 2" class="notEnoughNodesError alert alert-danger mt-1">
+                                            {{'cluster_management.cluster_page.errors.at_least_two_nodes' | translate}}
+                                        </div>
+                                    </div>
+                                </div>
+                                <ul ng-if="clusterConfiguration.nodes && clusterConfiguration.nodes.length" class="list-group nodesList">
+                                    <li ng-repeat="node in clusterConfiguration.nodes" class="list-group-item list-group-item-action d-flex justify-content-between align-items-center node">
```

Comment:
trackBy

Reply:
It is just a list of strings for now. Will add trackBy if/when it becomes an object[]